src/main/resources/bridge-server.conf

```diff
@@ -0,0 +1,155 @@
+channel.throttle.max.requests = 1
+channel.throttle.timeout.seconds = 300
```

Comment:
Why are we duplicating this file? It's error-prone to have to make the same changes in both projects. We should have both projects reading from the conf file.

Reply:
Removed